nss_cache/sources/ldapsource.py

```diff
@@ -586,11 +602,17 @@ def Transform(self, obj):
     gr.passwd = '*'
     members = []
     if 'memberUid' in obj:
-      members.extend(obj['memberUid'])
+      if hasattr(self, 'groupregex'):
```

Comment:
```python
if getattr(self, 'groupregex', None):
```
`getattr` doesn't have a default... by default :sunglasses: but can be supplied one.